presto-main/src/main/java/com/facebook/presto/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -166,21 +163,18 @@ public static RowExpression translate(
     private static class Visitor
             extends AstVisitor<RowExpression, Void>
     {
-        private final FunctionKind functionKind;
```

Comment:
Removing FunctionKind should in previous commit (Convert CallExpression to use FunctionHandle)?